### DISCUSSION TITLE
Organic Maps Ambassadors

### DISCUSSION THREAD

**Discussion by rtsisyk** [organicmaps/organicmaps#5270] [Announcements]
Hello Community! We are delighted to see the growing recognition and attention that Organic Maps is receiving, thanks to your invaluable contributions and efforts.  Together, we have accomplished what was previously hard to believe.

Today, we are excited to introduce the new **Organic Maps Ambassadors** initiative. Building upon the achievements of our successful [i18n efforts](https://github.com/orgs/organicmaps/discussions/4515), this new program aims to our community engagement and outreach efforts.

🍃 Who are **Organic Maps Ambassadors**? We are looking for initiative and active people who can drive community building efforts in their regions & languages. Ambassadors will be the face and voice of Organic Maps in their region and their language, engaging with users, coordinating community efforts, and advocating for the project's goals and values. Responsibilities are including but not limited to facilitating discussions, encouraging collaboration, and promoting knowledge-sharing among community members. Ambassadors will be spreading the word about Organic Maps by all possible means and communication channels, including forums, mailing lists, social media, and online platforms.

🗺️ Regions & languages to cover (with no particular order) :

- Germany – German
- France – French
- Austria – German
- Switzerland – German, French
- Spain – Spanish
- Italy – Italian
- Poland – Polish
- China – Mandarin
- United States, Canada  – English
- Australia, NZ – English
- United Kingdom – English
- South America – Portuguese, Spanish
- India – English, Hindi, and other languages 
- Pakistan – English, Urdu, and other languages
- Gulf Area – Arabic
- South East Asia – Vietnamese, Thai, Malay, and others
- ... 

🌐 This is list is **open**, please feel to create community in your area & language. Global worldwide efforts in English are welcomed too. The Organic Maps project is committed to supporting and valuing all contributions in this domain. We will strive to provide the necessary resources, including media, engineering, and financial support, to the best of our ability. You will undoubtedly gain valuable experience, along with recognition and respect from the open-source community.

✉️ Please drop reply here or drop an email to hello@organicmaps.app to start the conversation. 

CC: @organicmaps/translations

**Comment by Jean-BaptisteC** [comment]
Great idea, i'm don't have enough time to become French ambassadors but i want to continue to test OM, open issues about bugs and crash and answer to questions on telegram channel.
To communicate this discussion at the most of people, maybe send mail to people of https://weeklyosm.eu
The next week-end in Marseille, France we have [SOTM2023](https://sotm2023.openstreetmap.fr/), i will try to communicate if any people are interested to become ambassadors

**Comment by MetehanOzyurek** [comment]
I'm already helping with Turkish support chat on Telegram and posting new updates on Instagram. I've been busy for a while but now I can continue to keep the existing Turkish accounts active and help with email support and responding to store app comments. I am ready to help you in Turkish.

**Reply by rtsisyk** [reply]
This is why Turkey is not mentioned in the list ;)

**Reply by MetehanOzyurek** [reply]
Ah ok😅

**Reply by orkutmuratyilmaz** [reply]
I'd be glad to help @MetehanOzyurek. Can we put Turkey to the list now? 😀

**Reply by MetehanOzyurek** [reply]
> I'd be glad to help @MetehanOzyurek. Can we put Turkey to the list now? 😀

I think, some help would be great. @rtsisyk can you also add @orkutmuratyilmaz to translation-tr team?

**Reply by orkutmuratyilmaz** [reply]
Thank you @MetehanOzyurek for recommending.

**Reply by rtsisyk** [reply]
@orkutmuratyilmaz hoşgeldin

**Reply by orkutmuratyilmaz** [reply]
thank you and hoşbuldum:)

**Comment by Mr-Mime** [comment]
I could help with the german speaking countries (germany, austria & switzerland)

**Reply by j13m126** [reply]
I'd like to volunteer as well. For Austria or ASG in general.

**Comment by Chisom1010** [comment]
Is Nigeria among the countries where ambassadors are needed?

**Reply by rtsisyk** [reply]
Absolutely!

**Reply by Chisom1010** [reply]
Great, I am glad and willing to help with Nigeria.

**Comment by kshitijrajsharma** [comment]
Hi ! Great idea 💡 I can help for India , Nepal Region !

**Reply by rtsisyk** [reply]
I think that we are already talking in email.

**Comment by thatlindseygirl** [comment]
Hi, I'd be glad and willing to help with West Africa and in English! If country-specific, then Ghana!

**Reply by rtsisyk** [reply]
Wow, nice to hear from Ghana! Would you mind dropping a message to [hello@organicmaps.app](mailto:hello@organicmaps.app)?

**Reply by thatlindseygirl** [reply]
Yes sir! Done that, thanks.

**Comment by Supaplextw** [comment]
I can help Taiwanese users with Taiwanese Mandarin. If you need, I can use translation tools to support users using Simplified Chinese.

**Reply by rtsisyk** [reply]
you are already in the @organicmaps/translations-zh-hant team ;)

**Comment by SawanShariar** [comment]
I would be happy to join from Bangladesh

**Reply by rtsisyk** [reply]
Welcome!!! I am reading your website. Very cool! Please drop a message to hello@organicmaps.app once you are free.

**Comment by Kerenyambura** [comment]
Is Kenya among the countries needing an ambassador?

**Comment by patepelo** [comment]
It is a good idea to have regional ambassadors, specially to promote bidirectional collaboration, for example in representation of local differences. That it can be nourished into the app so that OM can have a product that is universal and democratic for each corner of the globe.

I was thinking maybe we could have ambassadors to niche groups, lice cyclists, but just a raw idea.
